LTTngCds/CookerData/LTTngContext.cs

```diff
@@ -3,6 +3,7 @@
 
 using System;
```

Comment:
Please bump versions in LTTngCds .csproj & LTTngDataExtensions .csproj

Reply:
Done